modules/project/service-agents.yaml

```diff
@@ -155,6 +155,7 @@
   service_agent: "service-%s@gcp-sa-datapipelines.iam.gserviceaccount.com"
 - name: "dataplex"
   service_agent: "service-%s@gcp-sa-dataplex.iam.gserviceaccount.com"
+  jit: true
```

Comment:
Please add role information here: `roles/dataplex.serviceAgent` (and also to [table in the README](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/tree/master/modules/project#service-identities-requiring-manual-iam-grants))